add-npm-geofilter.py

```diff
@@ -20,17 +21,17 @@ def ip_range_to_cidr(ip_start, ip_end):
 
 parser = argparse.ArgumentParser(description='Add npm geofilter')
 parser.add_argument('--npm-host', help='The npm host / ip to connect to', required=True)
-parser.add_argument('--npm-port', help='The npm port admin port', required=True)
-parser.add_argument('--npm-email', help='npm admin email address', required=True)
-parser.add_argument('--npm-password', help='npm admin password', required=True)
+parser.add_argument('--npm-port', help='The npm port admin port', required=False)
+parser.add_argument('--npm-email', help='npm admin email address', required=False)
+parser.add_argument('--npm-password', help='npm admin password', required=False)
 parser.add_argument('--npm-accesslist-name', help='The name of the access list', required=True)
 parser.add_argument('--allowed-countries', help='The list of allowed country codes',type=csv_list, required=True)
 parser.add_argument('--ip-list-file', help='The file containing the ip list', required=False)
 
 args = parser.parse_args()
 
 npmclient.BASE_URL = args.npm_host
-npmclient.BASE_PORT = args.npm_port
+if args.npm_port is not None: npmclient.BASE_PORT = args.npm_port

```

Comment:
default port is 81.. if it's not a required parameter it should be set to the default in the argument definition..

Reply:
ok yeah I forgot I put the default in the client.. been a while since I wrote this :)